### ISSUE TITLE
Female harpies models

### ISSUE THREAD

**Issue by Mechafromthe80s** [Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2#1137]
**Describe your suggestion in full detail below:**

Harpy in-game models

Mod team designers are very capable people and I believe they can make harpy models in game by altering existing elves models.
Just to test this idea, I tried to make something that looks like a harpy by using High Elf model as a base
The result is this -

![image](https://user-images.githubusercontent.com/127566238/233429987-5e2c7e40-e143-4b7c-a57d-01cbdf5d48f8.png)

Now, obviously this isn't perfect. The designer team can do better
And the model lacks certain features, like harpy-type eyes, harpy-type ears, harpy-type hair, harpy-type jewelry. Not to mention the feathers and harpy-type arms and legs -

![image](https://user-images.githubusercontent.com/127566238/233431062-ad2a1190-94d6-44e0-88f8-73423e97ea3b.png)

I also would like to show the concept with a harpy wearing armor. I think this is a neat idea, to have them wear something, at least from time to time - 

https://www.deviantart.com/iriadescent/art/Harpy-Concept-741206015

In short, I think bringing more in-game models to the game would be good and I am sure that the designers can make it happen.

**Comment by ercarp** [comment]
This looks excellent. You did a great job capturing the way they look.

We had planned to eventually™ add proper ethnicities for some of the more humanoid-looking creatures such as Harpies, Satyrs, and Centaurs, but as you mentioned, there are still quite a bit of 3D assets that need to be incorporated to fully realize an ethnicity like this using CK3's genes. We would need custom blendshapes for the ears and legs, but seeing as the draenei legs are basically halfway there, it wouldn't require as much work.

I can't give you an ETA of any kind, but I hope we can get around to adding more CK3 ethnicities to the mod soon.

**Comment by Mechafromthe80s** [comment]
Thank you!

Sounds great. I imagine centaurs would be hard to do.
Cool to hear that draenei legs are coming

Will be waiting for GoA2 getting better and better.
